cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -4,13 +4,19 @@ kind: Template
 metadata:
```

Comment:
What is the meaning of "integration" in this context? We are in the ARO-HCP repository, so I might expect integration to mean ARO-HCP Integration environment

Reply:
It is not for the integration env. It is for the CS PR Check environment. We should fix this

Reply:
Fix as in we create another directory in `deploy` and move this template there?